akka-actor/src/main/scala-3/akka/util/ByteIterator.scala

```diff
@@ -0,0 +1,699 @@
+/*
+ * Copyright (C) 2009-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
```

Comment:
all files in src/main/scala-3 are identical to src/main/scala-2.13

shall we create a symlink instead if the build tool doesn't understand something like scala-2.13+

Reply:
sbt can be made to understand 2.13+ in a few lines, here's how cats does it for example: https://github.com/typelevel/cats/blob/8a56313b4caccac6a28d877b54bc27bd08ef1a0d/build.sbt#L133-L142

Reply:
(also symlinks in git are generally a bad idea because of a bunch of complications when using Windows)

Reply:
Created #29991 to track fixing this